helmfile/charts/networkpolicy/common/templates/cert-manager/startupapicheck.yaml

```diff
@@ -11,6 +11,7 @@ spec:
   policyTypes:
     - Egress
   egress:
```

Comment:
Can you check if an empty egress results in `egress: {}`? If that is the case you need to move the if state

Reply:
> Can you check if an empty egress results in `egress: {}`? If that is the case you need to move the if state

Checked this and an empty egress resulted in `egress:` not being present in the resulting policy